Minio/HttpRequestMessageBuilder.cs

```diff
@@ -172,6 +172,11 @@ public void AddQueryParameter(string key, string value)
         QueryParameters[key] = value;
     }
 
+    public void AddQueryParameter(string key, int value)
```

Comment:
Added to make it easier to add an integer value, doing `.ToString()` in one place.